android/walletlib/src/main/java/com/solana/mobilewalletadapter/walletlib/authorization/AuthDatabase.java

```diff
@@ -29,18 +35,35 @@ public void onConfigure(SQLiteDatabase db) {
     public void onCreate(SQLiteDatabase db) {
         db.execSQL(IdentityRecordSchema.CREATE_TABLE_IDENTITIES);
         db.execSQL(AuthorizationsSchema.CREATE_TABLE_AUTHORIZATIONS);
-        db.execSQL(PublicKeysSchema.CREATE_TABLE_PUBLIC_KEYS);
+        db.execSQL(AccountRecordsSchema.CREATE_TABLE_ACCOUNTS);
         db.execSQL(WalletUriBaseSchema.CREATE_TABLE_WALLET_URI_BASE);
     }
 
     @Override
     public void onUpgrade(SQLiteDatabase db, int oldVersion, int newVersion) {
```

Comment:
When does `onUpgrade` trigger? Or, in other words, how does walletlib know when to `onUpgrade` the database?

Reply:
onUpgrade is called whenever the database schema version changes (5->6 in this case)